packages/node/src/http.ts

```diff
@@ -76,7 +107,21 @@ export const Http: Plugin = (_options, exporter) => {
           return end.apply(this, args);
         };
 
+        // captures the response
         request.addListener('response', response => {
+          _timestamps.firstByte = process.hrtime();
+
+          // Now we know whether `lookup` or `connect` happened. It's possible they
+          // were skipped if the hostname was already resolved (or we were given an
+          // IP directly), or if a connection was already open (e.g. due to
+          // `keep-alive`).
+          if (!_timestamps.lookup) {
```

Comment:
I think there may be a bug here (and wherever something from `_timestamps` is used as a condition), since they're initialized to `[0, 0]` which will be truthy.